src/header/header.scss

```diff
@@ -274,61 +227,72 @@
 
   .iui-header-button-icon {
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
   }
 
+  img.iui-header-button-icon {
+    padding: 0;
+    width: auto;
+    height: $iui-icons-default + ($iui-xs * 2);
+  }
+
   &.iui-active {
     @include iui-header-button-active;
   }
 
-  &:disabled {
+  &.iui-disabled,
+  &:disabled,
+  &[aria-disabled='true'] {
     .iui-header-button-icon {
       filter: grayscale(1) opacity(0.7);
     }
   }
 
   @at-root {
-    .iui-page-header.iui-slim {
-      > .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            width: $iui-icons-small;
-            height: $iui-icons-small;
-            padding: $iui-xxs;
-          }
+    .iui-slim {
+      .iui-header-breadcrumbs-button {
+        .iui-header-button-icon.iui-header-button-icon {
+          width: $iui-icons-small;
+          height: $iui-icons-small;
+          padding: $iui-xxs;
         }
 
-        .iui-button-label {
-          .iui-description {
-            @include iui-display-none;
-          }
+        img.iui-header-button-icon.iui-header-button-icon {
+          width: $iui-icons-small + ($iui-xxs * 2);
+          height: $iui-icons-small + ($iui-xxs * 2);
+          padding: 0;
+        }
+
+        .iui-description {
+          @include iui-display-none;
         }
       }
     }
   }
 }
 
 @mixin iui-header-split-button {
+  max-width: 25vw;
```

Comment:
do you remember the story behind this value?

Reply:
I think we needed to set a width for truncation to take place and we landed on 25% viewport width. 🤷🏻‍♂️